Add Advanced Processing to Logs Supplementary Guidelines (#4407)
open-telemetry/opentelemetry-specification: specification/logs/sdk.md

  +-----+------------------------+
```

> Please see [Supplementary Guidelines](./supplementary-guidelines.md#complex-processing)

Author: ```suggestion
Please see [Supplementary Guidelines](./supplementary-guidelines.md#advanced-processing)
```

> for more information on how to setup complex log record processing

Author: ```suggestion
for more information on how to setup advanced log record processing
```

> Please see [Supplementary Guidelines](./supplementary-guidelines.md#complex-processing)
> for more information on how to setup advanced log record processing
> (filtering, fan-out, routing).

Author: ```suggestion
Please see [Supplementary Guidelines](./supplementary-guidelines.md#advanced-processing)
for more information on how to setup advanced log record processing
(filtering, multiple pipelines, routing).
```
